src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts

```diff
@@ -79,7 +80,7 @@ export class DotHttpErrorManagerService {
         );
     }
 
-    private isLicenseError(response: Response): boolean {
+    private isLicenseError<T>(response: HttpResponse<T>): boolean {
```

Comment:
T here are just placeholders I think, we are not passing anywhere.

Reply:
https://github.com/dotCMS/core-web/pull/1429/files#r493928309

but on called on line
https://github.com/dotCMS/core-web/blob/d4202844f4f10011f925b09690dce7df8e36667d/src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts#L69